java-checks/src/main/resources/org/sonar/l10n/java/rules/java/Sonar_way_profile.json

```diff
@@ -275,6 +275,7 @@
     "S3252",
     "S3281",
     "S3305",
+    "S3329",
```

Comment:
Did you check why this issue has been added to the default profile?

Reply:
This rule has been recently implemented in C# in [MMF-2165](https://jira.sonarsource.com/browse/MMF-2165), and all rules are part of SonarWay. So it should make sense to have it part of it as well in Java.

Reply:
In fact, my concern would be that this rule is not behaving "well enough" to be included by default. Since it's a vulnerability, can @pierre-loup-tristant-sonarsource have a quick look at it? [RSPEC-3329](https://jira.sonarsource.com/browse/RSPEC-3329)

Reply:
I was not aware this rule was not part of Sonar Way when I specified it a few month back.
When I look at RSPEC activity it looks like it always have been. I wonder if it was intended of not.
I had a quick look on the results of the rules on Peach and they look fine. 
I confirmed that it should be active by default.

Reply:
Thanks!